sentry_sdk/integrations/gcp.py

```diff
@@ -159,22 +159,22 @@ def _get_google_cloud_logs_url(initial_time):
     Returns:
         str -- Google Cloud Logs Console URL to logs.
     """
-    hour_ago = initial_time - timedelta(hours=1)
+    hour_ago = final_time - timedelta(hours=1)
 
     url = (
         "https://console.cloud.google.com/logs/viewer?project={project}&resource=cloud_function"
         "%2Ffunction_name%2F{function_name}%2Fregion%2F{region}&minLogLevel=0&expandAll=false"
-        "&timestamp={initial_time}&customFacets=&limitCustomFacetWidth=true"
-        "&dateRangeStart={timestamp_start}&dateRangeEnd={timestamp_end}"
-        "&interval=PT1H&scrollTimestamp={timestamp_current}"
+        "&timestamp={initial_time}Z&customFacets=&limitCustomFacetWidth=true"
+        "&dateRangeStart={timestamp_start}Z&dateRangeEnd={timestamp_end}Z"
+        "&interval=PT1H&scrollTimestamp={timestamp_current}Z"
     ).format(
         project=environ.get("GCP_PROJECT"),
         function_name=environ.get("FUNCTION_NAME"),
         region=environ.get("FUNCTION_REGION"),
-        initial_time=initial_time,
+        initial_time=final_time,
         timestamp_start=hour_ago,
-        timestamp_end=initial_time,
-        timestamp_current=initial_time,
+        timestamp_end=final_time,
+        timestamp_current=final_time,
```

Comment:
One of the advantages of using named keyword arguments to `str.format` is that you can reference the same value by name multiple times. Here, I believe there is no need to define both `timestamp_end` and `timestamp_current` when they always point to the same value.

In fact, `timestamp_current` is misleading because it is not the current time and it is used to set the `scrollTimestamp` value in the query string.

Suggestion: there should only be two timestamps passed to `format`, one for the start and the other for the end of the range.

Reply:
Made this change in the latest commit.

Reply:
Thanks!